integration-cli/docker_cli_build_test.go

```diff
@@ -5676,6 +5755,138 @@ func (s *DockerSuite) TestBuildContChar(c *check.C) {
 	c.Assert(result.Combined(), checker.Contains, "Step 2/2 : RUN echo hi \\\\\n")
 }
 
+func (s *DockerSuite) TestBuildCopyFromPreviousRootFS(c *check.C) {
```

Comment:
Other tests necessary on Windows:
- Case insensitivity in file/folder paths referenced in `COPY --from`
- Forbid `BUILD c:\ | c:\windows | WiNdows | c:\windows\`
- Allow `BUILD c:\file | c:\windows\file | c:\folder\file`
- Forbid `BUILD .` where working dir is either `c:\` or `c:\windows` (case insensitive)